test/modules/events.js

```diff
@@ -499,9 +499,9 @@
             assessGestureExpectations({
                 description:           'enter-nonprimarypress-nonprimaryrelease-exit (secondary/right click):  ',
                 enterCount:            1,
-                exitCount:             1,
+                exitCount:             2,
```

Comment:
Can you explain why we're getting more of these now? Should we be concerned about possible side effects?